src/themes/appletv/theme.css

```diff
@@ -293,7 +293,20 @@ html {
 
 .emby-checkbox:checked + span + .checkboxOutline,
 .itemProgressBarForeground {
-    background-color: #00a4dc;
+    background: rgb(0, 210, 201);
+    background: -moz-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
+    background: -webkit-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
+    background: linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
+}
+
+.itemProgressBar {
+    background: rgba(230, 230, 230, 0.8);
+    height: 0.18rem;
+}
+
+.innerCardFooter {
+    border-radius: 0.14rem;
+    margin: 6px 8px 6px 8px;
```

Comment:
Please avoid using `px` and prefer `em` or `rem`.

Reply:
Thanks, I will change